--- bzip2-1.0.6/Makefile-libbz2_so.old	2019-02-02 11:45:09.000000000 -0800
+++ bzip2-1.0.6/Makefile-libbz2_so	2019-02-02 11:45:17.000000000 -0800
@@ -35,7 +35,7 @@
       bzlib.o

 all: $(OBJS)
-	$(CC) -shared -Wl,-soname -Wl,libbz2.so.1.0 -o libbz2.so.1.0.6 $(OBJS)
+	$(CC) -shared -Wl,-install_name -Wl,libbz2.so.1.0 -o libbz2.so.1.0.6 $(OBJS)
 	$(CC) $(CFLAGS) -o bzip2-shared bzip2.c libbz2.so.1.0.6
 	rm -f libbz2.so.1.0
 	ln -s libbz2.so.1.0.6 libbz2.so.1.0
